data/net/minecraft/world/phys/AABB.mapping

```diff
@@ -1,4 +1,19 @@
 CLASS net/minecraft/world/phys/AABB
+	METHOD <init> (DDDDDD)V
+		ARG 1 minX
+		ARG 3 minY
+		ARG 5 minZ
+		ARG 7 maxX
+		ARG 9 maxY
+		ARG 11 maxZ
```

Comment:
While this is accurate in describing what a valid input may be to the constructor, it would be more accurate if the `min`/`max` prefixes are removed (and possibly replaced with `first`/`secnod`), since the constructor uses `Math.min` and `Math.max` to assign the parameters to the fields properly.